### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by Mekuseo**
- seperated my code into modules
-kept all functionalities of the code without errors
- Applied ES6 principles
- updated time with Luxon Library

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,7 +19,6 @@ <h3>Awesome Books</h3>
         </ul>
       </nav>
       <p id="currentDate"></p>
-
       <!-- Add a Book -->
       <section class="add-new-book">
           <h3 class="book-heading">Add a New Book</h3>
```
Comment (thecodechaser) [inline index.html]: - [x] The footer is only visible in the list section, Please make sure that the footer is displayed on all three sections of the app.

Comment (thecodechaser) [inline index.html]: - [x] After adding books to the collection If I remove books from the collection It gets removed but when I refresh the page, I still see those books on the collection. Please make sure to fix this issue. Removing books from the collection should also update the local storage and after the page refresh the removed books should not display on the collection.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Mekuseo👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️

- [ ] https://github.com/Mekuseo/Awesome-Book-Modules/blob/8aa49943dabbe2ab7db31339546d06e58d4c08b8/modules/book.js#L14-L46
  - Please change these functions to arrow functions to follow the ES6 syntax.

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Mekuseo) [reply index.html]: fixed

Comment (Mekuseo) [reply index.html]: fixed

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Mekuseo 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Well done on making the requested changes from the previous reviewer

**Optional**
- Please fix the MIT link for a professional README file

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**